src/SlackCommand.ts

```diff
@@ -0,0 +1,27 @@
+import { App, SlackCommandMiddlewareArgs, AwsLambdaReceiver } from '@slack/bolt';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const awsLambdaReceiver = new AwsLambdaReceiver({
+  signingSecret: process.env.SLACK_SIGNING_SECRET as string,
+});
+
+export const slackapp = new App({
+  token: process.env.SLACK_BOT_TOKEN as string,
+  receiver: awsLambdaReceiver,
+});
+
+slackapp.command('/bonus', async ({ command, ack, respond }: SlackCommandMiddlewareArgs) => {
+  await ack();
+  
+  const responseMessage = `Hello, <@${command.user_id}>! 👋`;
+  await respond(responseMessage); 
+  
+  console.log('Slack command executed:', {
+    userId: command.user_id,
+    command: '/bonus',
+    details: command,
+    timestamp: new Date().toISOString()
+  });
+});
```

Comment:
I would assume that @lockmhe meant removing the `export` keyword from all the "internal" stuff here and introducing something like:
```typescript
export async function slackMessageHandler(event: APIGatewayProxyEvent, context: Context, cb: Callback): Promise<AwsResponse> {
  // .. internals are home work
}
``` 
which would allow you to simplify the `HandleSlackEvent` to
```typescript
  console.log('Received event:', JSON.stringify(event, null, 2));

  try {
    return slackMessageHandler(event, context, callback);
  } catch (error) {
    console.error('Error processing the request:', error);
    return {
      statusCode: 500,
      body: JSON.stringify({ message: 'Internal server error' }),
    };
  }
```

Remember that the Lambda environment stays warm for some time. 
Calling `awsLambdaReceiver.start()` should be necessary only once when the lambda boots up